src/components/rockets/Rockets.jsx

```diff
@@ -1,5 +1,17 @@
-const Rockets = () => (
-  <div>Rockets go here</div>
-);
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { getRockets } from '../../redux/rockets/rockets';
+
+const Rockets = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getRockets);
+  }, []);
```

Comment:
Calling `useEffect` inside the component will be triggered every time the component is rendered. Even when you check if the state is empty or not before you fetch data, `useEffect` is still triggered on every render. A better way to do this is to call `useEffect` inside `app.js` which will be triggered only once when the app is rendered.

Reply:
https://stackoverflow.com/questions/59817725/react-is-it-bad-practice-to-use-multiple-api-calls-in-app-component